### ISSUE TITLE
Running clusterctl with existing infra generates a bad kubeconfig

### ISSUE THREAD

**Issue by chuckha** [kubernetes-sigs/cluster-api-provider-aws#348]
/kind bug

**What steps did you take and what happened:**
I created a cluster and then created another cluster. The kubeconfig gave me this error:

```
Unable to connect to the server: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "kubernetes")
```

**What did you expect to happen:**
I expected kubeconfig to connect to my *new* cluster

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api-provider-aws version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

**Comment by chuckha** [comment]
it does however reuse my old vpc but creates a new machine. Maybe this is expected but it doesn't behave very well.

**Comment by detiber** [comment]
I suspect part of this may be because we are storing the cert material in the status, which is wiped when the cluster-api components are pivoted.

**Comment by randomvariable** [comment]
Yup. Should I complete the SSM Parameter store proposal? I think the implementation was pretty trivial when I last did it.

**Comment by detiber** [comment]
@randomvariable sounds good, there is a temporary workaround to move the cert/key to config rather than status, but I think SSM provides a better long term story here.

**Comment by vincepri** [comment]
Is this fixed now that #355 is merged?

**Comment by vincepri** [comment]
Closing this as it should be fixed in #355, feel free to reopen if not.
/close
